tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -404,6 +404,113 @@ def test_lambda_code_location_s3(
             == get_function_response_updated["Configuration"]["CodeSize"]
         )
 
+    # TODO: fix type of AccessDeniedException yielding null
+    @markers.snapshot.skip_snapshot_verify(paths=["function_arn_other_account_exc..Error.Message"])
```

Comment:
We would need some other type of `AccessDeniedException` to fix this tiny snapshot diff because AWS returns `None` but the exception translates the None message into an empty string:

`/Error/Message None (type: <class 'NoneType'>) → '' (type: <class 'str'>)... (expected → actual)`